modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -1,317 +1,460 @@
-
-* CloudHub: When using CloudHub-Connector to trigger notifications and alerts, there is a limit of five custom application alerts per IP address, per second. If an application uses multiple workers, this limit is multiplied by the number of workers assigned to the application, since the limitation is enforced per IP. +
-Custom notifications that trigger emails are also subject to the rate limit on email notifications described below.
-* Alert emails: Both standard and custom alerts have a rate limit where only one alert notification email is sent every 2.5 seconds. This limit is enforced by a combination of organization, environment, alert (name/ID), and resource (application, server, server group, or cluster).
+Anypoint Platform enforces rate limits that determine how often an alert is triggered and prevent excessive emails from being sent.
+The following factors determine the rate limit:
 
+* CloudHub
++
+When using CloudHub Connector to trigger notifications and alerts,
+the rate limit is five custom application alerts per IP address, per second.
+This limit is enforced by IP address. 
+If an application uses multiple workers, this limit is multiplied by the number of workers
+assigned to the application.
++
+Custom notifications that trigger emails are also subject to the rate limit on email notifications.
+* Alert emails
++
+For both standard and custom alerts, the rate limit is one alert notification email sent every 2.5 seconds.
+This limit is enforced by a combination of organization, environment, alert (name/ID), and resource
+(application, server, server group, or cluster).
 
 == Create an Alert
 
-If you access the alerts screen through any of the paths listed above and you don't have any alerts set up yet, you will see the following screen:
+To create an alert:
 
-image::create-first-alert.png[first-alert]
+. In Runtime Manager, click *Alerts* in the left navigation pane.
++
+** If no alerts are set up, you see:
++
+.The arrow shows the *Create your first alert* option.
+image::alerts-no-alerts.png[Alerts page with no alerts set up]
+** If your account already has associated alerts, you see:
++
+.The arrow shows the blue plus (*+*) icon on the *Alerts* page.
+image::alerts-create-additional-alert.png[Alerts page with blue plus (+) icon]
 
-. Click the plus sign next to *Create your first alert*
+. Click the blue plus (*+*) icon to create an alert.
 +
-image::create-alert.png[]
+image::alerts-create-alert.png[Create an Alert page]
 
 . Fill in the required fields:
 
-* *Name* - Unique and meaningful name for the alert
-* *Severity Level* - (Critical, Warning or Info)
+* *Name*
 +
-[NOTE]
-Different severity levels currently have no effect on alerts, but will in the future when further features are released.
+Unique and meaningful name for the alert
+* *Severity Level*
```

Comment:
Is "level" init capped in the UI? Because later I see "Application type."

Reply:
Yes. See screenshots.

Reply:
Oops, nope. The capitalization is all over the place on this page.